docs: vignette on the rationale for the algorithm (#63)
steno-aarhus/osdc: vignettes/rationale.Rmd

    and how applying or using these approaches isn't very easy.
-   How this algorithm and package contributes to discussions in this
    space about how diabetes in classified in Danish register research
    and how it is implemented.

## Identifying type 1 and 2 diabetes cases in Danish healthcare registers

### Danish register data infrastructure

Many individual-level data (e.g. civil registration, public healthcare
contacts, and drug prescriptions) are automatically collected on all
residents in Denmark and stored in nationwide Danish registers by
[Statistics Denmark](https://www.dst.dk/da/) and the [Danish Health Data
Authority](https://sundhedsdatastyrelsen.dk/da). These agencies are
legally allowed to give access to the register data for research
purposes, which provides (authorized) researchers a set of common,
extensive data sources to use for studies. Any researcher associated
with an approved Danish research institute (mainly Danish universities)
can apply for access, but fees and conditions apply.

Register data is generally accessed and processed by approved
researchers on remote servers operated by Statistics Denmark and the

> Danish Health Data Authority. The same raw data used by all researchers
> and the common virtual working environment potentially enables
> reproducible research. This means that any data processing workflow

Reviewer: ```suggestion
Danish Health Data Authority. The same raw data used by all 
researchers, coupled with a common virtual working environment, 
has the potential to enable reproducible research. This means that 
any data processing workflow
```

> could be transferable and usable between research projects if the

Reviewer: ```suggestion
could be transferable and reusable between research projects if the
```

underlying code is designed with reproducibility in mind and the code is
shared ("open-sourced") [@Marszalek2016]. While reproducibility in
research relates to transparent reporting of methods to enable others to
reproduce analyses and experiments, this also applies to a diabetes
classification program, which - if reproducible - could be reused by any
researcher with access to the necessary register data to dynamically
identify a study population of individuals with diabetes for their
research needs [@Dima2017].

### Current Danish register-based diabetes classifiers

> In Denmark, the first resource readily available to researchers to
> identify diabetes cases through register data was the National Diabetes
> Register, which was established in 2006 [@Carstensen2011] but then
> discontinued in 2012.

Reviewer: ```suggestion
In Denmark, the National Diabetes Register, established in 2006, was the first resource readily
 available to researchers to use for identifying diabetes cases through register data [@Carstensen2011] .
However, it was discontinued in 2012.
```

> The next resource is the Register of Selected Chronic Diseases (RSCD),

Reviewer: Add source for RSCD? 
The Danish Health Data Authority. The register of selected chronic diseases; 2014. Available from: https://www.esundhed.dk/Dokumentation/DocumentationExtended?id=29. Accessed November 25, 2022.

> The next resource is the Register of Selected Chronic Diseases (RSCD),

Author: ```suggestion
The next resource is the [Register of Selected Chronic Diseases](https://www.esundhed.dk/Dokumentation/DocumentationExtended?id=29) (RSCD),
```

which was launched in 2014. It is currently the only publicly available
resource to identify diabetes cases through Danish register data (by
application to the Danish Health Data Authority).

## Challenges in current classifiers

General-purpose registers and other administrative databases often
provide the basis of diabetes epidemiology, but they rarely contain

> validated diabetes-specific data, which may introduce bias in studies

Reviewer: Maybe relevant to note what kinds of biases?
